app/Http/Controllers/AppController.php

```diff
@@ -99,7 +101,7 @@ public function loginUser(): RedirectResponse
         ]);
 
         // check if user exists
-        $user = User::where('email', $validated['email'])->first();
+        $user = User::where('email', strtolower($validated['email']))->first();
```

Comment:
This could lead to errors if a user is already present in the database with non lowercase emails. I think we have to add a migration that transforms all emails to lowercase. 

And maybe we have to update the seeders as well that force lowercase emails.